setup.py

```diff
@@ -13,11 +13,47 @@
 # governing permissions and limitations under the License.
 
 from setuptools import setup, find_packages
+from os.path import dirname, join
+import os, sys
+from os.path import exists
+readme_file = 'README.rst'
+
+def _get_readme():
+    with open(join(dirname(__file__), readme_file)) as f:
+        return f.read()
+
+hcfg = "/etc" if os.access('/etc/', os.W_OK) else ''
+daemon_dir = '/usr/lib/systemd/system' if exists('/usr/lib/systemd/system') \
+		and os.access('/usr/lib/systemd/system', os.W_OK)\
+		 else ('/lib/systemd/system' if os.access('/lib/systemd/system', os.W_OK) else '')
```

Comment:
You should *NOT* be trying to install these in setup.py; this is something that would happen at the level distro packages (rpm/deb etc).

Also, super important: in general, *do not* do this kind of autodetection from `setup.py`. It isn't safe to assume that the build machine is the same as the machine that will actually run the software. Even in python there *is* a build step, at least when you're doing full packaging of the software.

Just make sure the files are included by the MANIFEST, and leave it at that. If/when we make distro packages, we can deal with this there.